tests/samples_smoke_tests/README.md

```diff
@@ -0,0 +1,29 @@
+These test executes IE samples on pregenerated IR
+
+1. To run tests required installing some dependencies: 
+    - pip3 install -r requirements.txt
+
+2. Download archive with models and validation set: [samples tests data](https://d30ikxcvcet9xo.cloudfront.net/repositories/openvino/ci_dependencies/test/samples_smoke_tests_data.zip)
+3. Set environment variables:
+a. Required:
+```sh
+- IE_APP_PATH=<path to samples only for C++ and C samples>
+- IE_APP_PYTHON_PATH=<path to python IE samples only for python samples>
+- PYTHONPATH=<path to model-optimizer-tensorflow>:<path to samples_smoke_tests>$PYTHONPATH>
```

Comment:
we have no `model-optimizer-tensorflow` component for now

Reply:
Was removed